specifications/cloud_assembly.md

```diff
@@ -0,0 +1,398 @@
+`exactImageId`|`string`|An absolute reference to the published image version (`imageName@DIGEST`).
+`imageName`   |`string`|The full tagged image name (`imageName:tagName`).
+
+##### Example
+```json
+{
+  "type": "npm://@aws-cdk/aws-ecr.DockerImageDrop",
+  "environment": "aws://000000000000/bermuda-triangle-1",
+  "properties": {
+    "savedImage": "docker/37e6de0b24fa.tar",
+    "imageName": "${helperStack.ecrImageName}",
+    "tagName": "latest"
+  }
+}
+```
+
+### Example
+Here is an example the contents of a complete *Cloud Assembly* that deploys AWS resources:
+```
+☁️ my-assembly.cloud
+├─ manifest.json                                Cloud Assembly manifest
+├─ stacks
+│  ├─ PipelineStack.yml                         CloudFormation template
+│  ├─ ServiceStack-beta.yml                     CloudFormation template
+│  ├─ ServiceStack-beta.stack-policy.json       CloudFormation stack policy
+│  ├─ ServiceStack-prod.yml                     CloudFormation template
+│  └─ ServiceStack-prod.stack-policy.json       CloudFormation stack policy
+├─ docker
+│  └─ docker-image.tar                          Saved Docker image (docker image save)
+├─ assets
+│  └─ static-website                            Files for a static website
+│     ├─ index.html
+│     └─ style.css
+└─ signature.asc                                Cloud Assembly digital signature
+```
+
+#### `manifest.json`
+```json
+{
+  "schema": "cloud-assembly/1.0",
+  "drops": {
+    "PipelineStack": {
+      "type": "npm://@aws-cdk/aws-cloudformation.StackDrop",
+      "environment": "aws://123456789012/eu-west-1",
+      "properties": {
+        "template": "stacks/PipelineStack.yml"
+      }
+    },
+    "ServiceStack-beta": {
+      "type": "npm://@aws-cdk/aws-cloudformation.StackDrop",
+      "environment": "aws://123456789012/eu-west-1",
+      "properties": {
+        "template": "stacks/ServiceStack-beta.yml",
+        "stackPolicy": "stacks/ServiceStack-beta.stack-policy.json",
+        "parameters": {
+          "image": "${DockerImage.exactImageId}",
+          "websiteFilesBucket": "${StaticFiles.bucketName}",
+          "websiteFilesKeyPrefix": "${StaticFiles.keyPrefix}",
+        }
+      }
+    },
+    "ServiceStack-prod": {
+      "type": "npm://@aws-cdk/aws-cloudformation.StackDrop",
+      "environment": "aws://123456789012/eu-west-1",
+      "properties": {
+        "template": "stacks/ServiceStack-prod.yml",
+        "stackPolicy": "stacks/ServiceStack-prod.stack-policy.json",
+        "parameters": {
+          "image": "${DockerImage.exactImageId}",
+          "websiteFilesBucket": "${StaticFiles.bucketName}",
+          "websiteFilesKeyPrefix": "${StaticFiles.keyPrefix}",
+        }
+      }
+    },
+    "DockerImage": {
+      "type": "npm://@aws-cdk/aws-ecr.DockerImageDrop",
+      "environment": "aws://123456789012/eu-west-1",
+      "properties": {
+        "savedImage": "docker/docker-image.tar",
+        "imageName": "${PipelineStack.ecrImageName}"
```

Comment:
According to your schema of values returned by `StackDrop` this ought to be `${PipelineStack.outputs.ecrImageName}`.

Reply:
Oh yeah. That might need to change. Good catch!